hologram_dashboard/fetcher/hologram_data_source_availability_fetcher.cc

```diff
@@ -20,6 +20,11 @@ namespace wireless_android_play_analytics{
 
 void HologramDataSourceAvailabilityFetcher::Process() {
     // TODO(alexanderlin): add implementation.
+    assert(FLAGS_config_file_path is not empty)
+    // AcquireConfig(FLAGS_config_file_path)
```

Comment:
AcquireConfig() should be called in constructor, since we only need to init hologram_config_ once.